components/blitz/resources/omero/api/ILdap.ice

```diff
@@ -16,26 +16,163 @@ module omero {
 
     module api {
         /**
-         * See <a href="http://downloads.openmicroscopy.org/latest/omero5.2/api/ome/api/ILdap.html">ILdap.html</a>
-         **/
+         * Administration interface providing access to admin-only
+         * functionality as well as JMX-based server access and selected user
+         * functions. Most methods require membership in privileged
+         * {@link omero.model.ExperimenterGroup} groups.
+         *
+         * Methods which return {@link omero.model.Experimenter} or
+         * {@link omero.model.ExperimenterGroup} instances fetch and load all
+         * related instances of {@link omero.model.ExperimenterGroup} or
+         * {@link omero.model.Experimenter}, respectively.
+         */
         ["ami", "amd"] interface ILdap extends ServiceInterface
             {
+                /**
+                 * Searches all {@link omero.model.Experimenter} list on LDAP
+                 * for attribute objectClass = person.
+                 *
+                 * @return all Experimenter list.
+                 */
                 idempotent ExperimenterList searchAll() throws ServerError;
+
+                /**
+                 * Searches Distinguished Name in groups
+                 *
+                 * @param attr   Name of member attribute. Never null or empty.
```

Comment:
This param marker is spilling into the short comment for this method:

https://ci.openmicroscopy.org/view/DEV/job/OMERO-DEV-merge-build/ICE=3.5,jdk=8_LATEST,label=octopus/javadoc/slice2html/omero/api/ILdap.html#ILdap

this is happening for most of the methods here, part of the first param is appearing with the @param being explicitly quoted.